docs/Reference/CLI/options.md

```diff
@@ -83,7 +83,7 @@ data-path: "/Users/me/my_node/data"
 
 Directory in which to store temporary files.
 
-### `key-store-path`
+### `key-store-path`, `key-config-path`
```

Comment:
key-config-path is the preferred name so maybe this should come first?

Reply:
I'm not sure updating the title is the right thing either. Compare to other options with an alias... https://doc-web3signer-git-fork-bgravenorst-web-185-infura-web.vercel.app/Reference/CLI/options#logging

Reply:
I'll make `key-config-path` first. We do the aliases for `--logging | -l`, `--version | -V`, and '--help | -h` differently due to the single dash (-) used by the alias. 

But generally we update the title to display the aliases. For example, in [Teku](https://docs.teku.consensys.net/reference/cli#data-base-path-data-path).

Reply:
On a seperate note, shouldn't we also create an alias for `--key-store-config-file-max-size`?

Reply:
Thanks @bgravenorst that makes sense. Yes, maybe we should create that alias too, good spot, I'll bring it up with the team